web/client/src/components/validation/TokenValidation.jsx

```diff
@@ -120,33 +132,50 @@ export default function ValidateToken() {
               container
               direction="row"
               justify="space-between"
-              alignItems="flex-start"
+              alignItems="flex-end"
             >
               <Grid item xs>
                 <Typography variant="h4">Validate Token</Typography>
               </Grid>
             </Grid>
           </div>
           <div>
-            <Grid
-              container
-              alignItems="flex-start"
-              justify="flex-end"
-              direction="row"
-            >
+            <Grid container alignItems="flex-end" justify="flex-end">
               {/* Indicated whether the token is valid or not. */}
               {completed && valid && (
                 <div>
                   {" "}
-                  <CheckCircleIcon color="primary" />
-                  <IconButton className="button-info" onClick={getTokenInfo}>
+                  <IconButton
+                    className="button-info"
+                    disabled
+                    classes={{ label: classes.iconButton }}
+                  >
+                    <CheckCircleIcon style={{ color: green[500] }} />
+
+                    <Typography variant="caption" style={{ color: green[500] }}>
```

Comment:
What does 500 indicate?

Reply:
The particular type of green from the green package.

Reply:
Cool!